subprojects/docs/docs.gradle

```diff
@@ -402,6 +402,30 @@ samples {
                 from(sampleDirectory.dir("common"))
             }
             // TODO (donat) sanity checks probably don't do anything for samples located one level down in the gradle/kotlin directories
+        }*/
+
+        compositeBuildsBasics {
+            sampleDirectory = file("src/samples/build-organization/compositeBuilds/basic")
```

Comment:
Instead of using `file(...)`, use `samplesRoot.dir("...")` this makes the path relative to `src/samples`.